chemprop/cli/train.py

```diff
@@ -791,23 +819,26 @@ def main(args):
         weight_col=args.weight_column,
         bounded=args.loss_function is not None and "bounded" in args.loss_function,
     )
+    if args.features_generators is not None:
+        # TODO: MorganFeaturizers take radius, length, and include_chirality as arguements. Should we expose these through the CLI?
+        features_generators = [
+            Factory.build(MoleculeFeaturizerRegistry[features_generator])
+            for features_generator in args.features_generators
+        ]
```

Comment:
In #748 we want to move the molecule featurizers out of datapoint and into the users hands. For the CLI this will mean changing making the molecule features and then passing them to the datapoints. Because this will be its own PR, I'll leave this as is for now and plan to incorporate your comment in that PR.